src/main.js

```diff
@@ -101,3 +107,10 @@ function checkDuplicate () {
   }
   return false;
 }
+
+function removePoster(target){
+  console.log(savedPosters);
```

Comment:
** **Note to self** ** remove console log

Reply:
Yes, definitely make sure these get pulled out _before_ the PR is merged 🙂